### PR TITLE
LP-2991 Extract strings from msp-assessment and ora

### PR DESCRIPTION
**Pull request by asadali145**
[LP-2991](https://philanthropyu.atlassian.net/browse/LP-2991)

***

#### Description:

- Extract xblock translations.

***

## Things to Cross-check:

- [ ] Technical Docs updated if needed?
- [ ] [Domain Model](https://drive.google.com/file/d/1iRSqxDQQMXl6JzLw9nZtq1yGshwMBLTn/view?usp=sharing) updated if needed?
- [ ] [Core Changes Wiki](https://github.com/OmnipreneurshipAcademy/edx-platform/wiki/Core-Changes) updated if needed?

### CODE REVIEW

File: .github/workflows/generate_translations.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
           python3 -m pip install pip==20.0.2
           python3 -m pip install setuptools==50.3.0
           python3 -m pip install wheel==0.35.1
-          git clone https://asadali145:${ASAD_PAT}@github.com/OmnipreneurshipAcademy/edx-msp-assessment-xblock.git@${ADG_XBLOCKS_BRANCH}#egg=msp-assessment-xblock
+          git clone https://${ASAD_PAT}@github.com/OmnipreneurshipAcademy/edx-msp-assessment-xblock.git@${ADG_XBLOCKS_BRANCH}#egg=msp-assessment-xblock
```
Comment (emraan004) [inline .github/workflows/generate_translations.yml]: Why there is ASAD_PAT here?

**Review by emraan004 — CHANGES_REQUESTED** [review]

File: .github/workflows/generate_translations.yml [hunk 2]
```diff
@@ -28,91 +28,97 @@ jobs:
       - name: Install pip requirements
         env:
           ADG_XBLOCKS_BRANCH: ${{ github.base_ref }}
+          ADG_PAT: ${{ secrets.ADG_PAT }}
+          ASAD_PAT: ${{ secrets.ASAD_PAT }}
```
Comment (emraan004) [inline .github/workflows/generate_translations.yml:32]: ASAD_PAT?

**Review by emraan004 — CHANGES_REQUESTED** [review]

File: .github/workflows/generate_translations.yml (hunk 2 above)
Comment (asadali145) [reply .github/workflows/generate_translations.yml:32]: Yes, Facing a few issues so trying a few things thats why used my own PAT.

File: .github/workflows/generate_translations.yml (hunk 1 above)
Comment (asadali145) [reply .github/workflows/generate_translations.yml]: Please ignore, Testing a few things as requirements are not being installed.

**Pull request closed without merging**